smartsim/experiment.py

```diff
@@ -380,7 +388,7 @@ def sprint(p):
             print(p, flush=True)
 
         sprint("\n")
-        models, ensembles, orchestrator = separate_entities(args)
+        models, ensembles, orchestrator, ray_clusters = separate_entities(args)
```

Comment:
I can see this getting possibly unruly if we add more object types, should ``separate_entities`` return a usable data structure instead? Something easy to add to later on? 

Something like

```python

class Deployment:

    def __init__(self, args):

   def get_entities():
        #seperates and returns entities
 
   def get_db():
         # ...

   and so on 

```

Advantage would be to have a data structure to be carried through the controller until jobs are created. The same data structure could be reused in other places/functions that require entity separation due to type agnostic APIs.